src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.html

```diff
@@ -28,7 +28,7 @@
          <div class="upload-button">
              <umb-button
                 type="button"
-                key="general_upload"
+                label-key="general_upload"

```

Comment:
I didn't know about this behavior, but noticed in other cases the 'label-key' attribute, when simply 'key' was here, so I just corrected this attribute, and leaved the 'label'...